openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ClassificationRepository.java

```diff
@@ -121,9 +121,8 @@ public ClassificationUpdater(
     @Transaction
     @Override
     public void entitySpecificUpdate() {
-      // TODO mutuallyExclusive from false to true?
       recordChange(
```

Comment:
@mohityadav766 lets remove the recordChange here, which should ignore any changes via patcha and we should also block the PUT calls to update it

Reply:
@harshach making it recordCHange original to original will keep this as it is in PUT and PATCH both